dcpy/connectors/edm/publishing.py

```diff
@@ -66,39 +67,45 @@ def get_source_data_versions(
 
 
 def upload(
-    output: Path,
```

Comment:
looks like this fn is used in factfinder as well. Do we need to change the keyword args there as well?

Reply:
Ah - will do. Thanks for catching, I didn't think these were used yet but I should have checked